### PR TITLE
Update the Codú challenge banner

### PR DESCRIPTION
**Pull request by NiallJoeMaher**
## Pull Request details

- Fix the col layouts so they are consistent on feed and homepage.
- Remove image that is no longer used

## Any Breaking changes

- Nope

## Associated Screenshots
New
New challenge banner with gradient
Old
Old banner that I thought looked poop

### CODE REVIEW

File: e2e/articles.spec.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ test.describe("Unauthenticated Articles Page", () => {
     ).toBeVisible({ visible: !isMobile });
 
     await expect(
-      page.getByRole("link", { name: '"Codú Writing Challenge" text' }),
+      page.getByRole("link", { name: '"Find out more" text' }),
```
Comment (JohnAllenTech) [inline e2e/articles.spec.ts]:
```suggestion
      page.getByRole("link", { name: "Find out more" }),
```

File: e2e/articles.spec.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ test.describe("Authenticated Articles Page", () => {
...
```
Comment (JohnAllenTech) [inline e2e/articles.spec.ts]:
```suggestion
      page.getByRole("link", { name: "Find out more" }),
```

File: e2e/articles.spec.ts [hunk 3]
```diff
@@ -82,7 +82,7 @@ test.describe("Authenticated Articles Page", () => {
...
```
Comment (JohnAllenTech) [inline e2e/articles.spec.ts]:
```suggestion
      page.getByRole("link", { name: "Find out more" }),
```

**Review by JohnAllenTech — CHANGES_REQUESTED** [review]
The changes I suggested will fix your tests.

image

Tested on your branch locally to confirm

File: e2e/articles.spec.ts (hunk 3 above)
Comment (NiallJoeMaher) [reply e2e/articles.spec.ts]: Thanks! Been a while since I touched Playwright and it shows 😂 Going to play with it to make sure I get gud!

**Pull request merged**